### ISSUE TITLE
Where to put the training or testing dataset? 

### ISSUE THREAD

**Issue by JiajiaLi04** [hustvl/BMaskR-CNN#22]
Dear Author,

I wonder where to put the training and testing dataset? Are there any rules to name them.   It is weired that you did not have the parameters in your parameter lines. Do I need to change the dataset name in the code?  Hoping for your reply. Thanks!

**Comment by wmrenr** [comment]
Did you solve this problem please?

**Comment by wondervictor** [comment]
Hi @JiajiaLi04 @wangxueping78, I'm much sorry for the late reply!
Preparing the dataset is simple, and you can refer to [Dataset for Detectron2](https://detectron2.readthedocs.io/en/latest/tutorials/builtin_datasets.html) for more details.
Exactly, if you MS-COCO dataset, you just download the dataset from the [official site](https://cocodataset.org/) and unzip the files.
Then you can enter into the directory of this repo by:
```bash
cd BMaskR-CNN
cd projects/BMaskR-CNN
```
setup a `datasets` folder, or use a soft link if you store the data in other place,
```bash
mkdir datasets
cd datasets
mkdir coco
```
then put the COCO dataset into `coco` directory, and then the directory tree of `projects/BMaskR-CNN` is like:
```bash
datasets
| -- coco
      | -- annotations
      | -- val2017
      | -- train2017
```
If you have any question, I'll reply as soon as possible.

**Comment by wmrenr** [comment]
> Hi @JiajiaLi04 @wangxueping78, I'm much sorry for the late reply! Preparing the dataset is simple, and you can refer to [Dataset for Detectron2](https://detectron2.readthedocs.io/en/latest/tutorials/builtin_datasets.html) for more details. Exactly, if you MS-COCO dataset, you just download the dataset from the [official site](https://cocodataset.org/) and unzip the files. Then you can enter into the directory of this repo by:
> 
> ```shell
> cd BMaskR-CNN
> cd projects/BMaskR-CNN
> ```
> 
> setup a `datasets` folder, or use a soft link if you store the data in other place,
> 
> ```shell
> mkdir datasets
> cd datasets
> mkdir coco
> ```
> 
> then put the COCO dataset into `coco` directory, and then the directory tree of `projects/BMaskR-CNN` is like:
> 
> ```shell
> datasets
> | -- coco
>       | -- annotations
>       | -- val2017
>       | -- train2017
> ```
> 
> If you have any question, I'll reply as soon as possible.
Oh thank you so much!
Excuse me,should the instances_train2017.json  and instances_val2017.json  be placed in the  annotations folder?

**Comment by wondervictor** [comment]
@wangxueping78 yep, the `instances_train2017.json` and `instances_val2017.json` should be placed into the `annotations` directory, that is,
```bash
datasets
| -- coco
      | -- annotations
            | -- instances_train2017.json
            | -- instances_val2017.json
      | -- val2017
      | -- train2017
```

**Comment by wmrenr** [comment]
oh，I get it！Thanks



发自我的iPhone


------------------ Original ------------------
From: Tianheng Cheng ***@***.***&gt;
Date: Thu,Mar 31,2022 11:12 PM
To: hustvl/BMaskR-CNN ***@***.***&gt;
Cc: wangxueping78 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [hustvl/BMaskR-CNN] Where to put the training or testing dataset? (#22)





 
@wangxueping78 yep, the instances_train2017.json and instances_val2017.json should be placed into the annotations directory, that is,
 datasets | -- coco       | -- annotations             | -- instances_train2017.json             | -- instances_val2017.json       | -- val2017       | -- train2017
